plugins/ping.rb

```diff
@@ -23,9 +32,39 @@ def match?(cmd)
     cmd =~ /^(!)?ping$/
   end
 
-  match /ping$/, method: :ping
+  def format_ping_time
+    milis = (Time.now - @timestamp) * 1000.0
+    mins = (milis / 60000).to_i
+    secs = (milis / 1000).to_i
+    str = ""
+    if mins > 0
+      str = "#{str}#{mins}"
+      str = "#{str}#{mins == 1 ? " minute, " : " minutes, "}"
+    end
+    if secs > 0
+      str = "#{str}#{secs}"
+      str = "#{str}#{secs == 1 ? " second, " : " seconds, "}"
+    end
+    return "#{str}#{((milis%60000)%1000).to_i} milliseconds"
```

Comment:
Don't use an explicit `return`; ruby will implicitly return the last expression.

Reply:
should I still assume milliseconds will always be millisecond**s** or do millisecond(s) also?